source/style-guide/writing/write-for-accessibility.txt

```diff
@@ -35,7 +35,7 @@ Writing with accessibility in mind improves the documentation experience for all
 Provide Context for UI Elements
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Do not identify UI elements using only one sensory characteristic (such
+Don't identify UI elements using only one sensory characteristic (such
```

Comment:
**note:** This style guide uses *GUI* and *UI*. May need to revisit this later across the style guide..

Reply:
[Noted](https://docs.google.com/document/d/1gwg44VP4_P-hDn-QhLEDOpon-ZnzfEFLxmIa4qy1kgQ/edit?usp=sharing).